test/mockHttpCalls/capgv1beta1/gcpClusters.ts

```diff
@@ -0,0 +1,101 @@
+import * as capgv1beta1 from 'model/services/mapi/capgv1beta1';
+
+export const randomGCPCluster1: capgv1beta1.IGCPCluster = {
+  apiVersion: 'infrastructure.cluster.x-k8s.io/v1beta1',
+  kind: 'GCPCluster',
+  metadata: {
+    creationTimestamp: '2022-07-12T06:19:52Z',
+    finalizers: [
+      'gcpcluster.infrastructure.cluster.x-k8s.io',
+      'dns-operator-gcp.finalizers.giantswarm.io',
+      'capg-firewall-rule-operator.finalizers.giantswarm.io',
+    ],
+    generation: 2,
+    labels: {
+      app: 'cluster-gcp',
+      'app.kubernetes.io/version': '0.15.1',
+      'cluster.x-k8s.io/cluster-name': 'uufda',
+      'giantswarm.io/cluster': 'uufda',
+      'giantswarm.io/organization': 'org1',
+    },
+    name: 'uufda',
```

Comment:
Just a suggestion, could we use this GCPCluster as the provider cluster for `randomClusterGCP1`? We don't use it for the tests here, but it might be helpful for future tests where we use both.

Also feel free to merge it as is, we can always change it as we need in the future 🙂